package Heart.proto;

message PackageEntry {
	optional string entryName = 1;
	optional uint64 entryOffset = 2;
	optional uint64 entrySize = 3;
	optional string entryType = 4;
}

message PackageHeader {
	repeated string			packageDependencies = 1;
	repeated PackageEntry	entries = 2;
}